compressor/management/commands/compress.py

```diff
@@ -148,7 +148,7 @@ def compress(self, log=None, **options):
             raise OfflineGenerationError("No template paths found. None of "
                                          "the configured template loaders "
                                          "provided template paths. See "
-                                         "http://django.me/template-loaders "
+                                         "https://docs.djangoproject.com/en/1.8/topics/templates/ "
```

Comment:
Should we use that link, or https://docs.djangoproject.com/en/1.8/ref/templates/api/#loading-templates ? or https://docs.djangoproject.com/en/1.8/ref/templates/api/#loader-types ? Up to you.